spec/factories/birth_record.rb

```diff
@@ -11,5 +11,17 @@
     parent_family_name { Faker::Name.last_name }
     other_parent_first_and_middle_names { Faker::Name.first_name }
     other_parent_family_name { Faker::Name.last_name }
+    trait :static_values do
+      # static values for Percy tests that compare visual diffs
+      first_and_middle_names { Timmy }
+      family_name { Target-Person }
+      date_of_birth { 1979-01-01 }
```

Comment:
i think this needs to be in quotes, so that it doesn't become a mathematics equation.
1979 - 01 - 01

Reply:
Indeed, forgot to put the quotes in all of these. Oops